packages/api/src/authentication/authentication.module.ts

```diff
@@ -0,0 +1,27 @@
+import { Module } from "@nestjs/common";
+import { AuthenticationService } from "./authentication.service";
+import { AuthenticationController } from "./authentication.controller";
+import { LocalStrategy } from "@cryptify/api/src/authentication/local.strategy";
+import { UsersModule } from "@cryptify/api/src/users/users.module";
+import { PassportModule } from "@nestjs/passport";
+import { JwtModule } from "@nestjs/jwt";
+import { JwtStrategy } from "@cryptify/api/src/authentication/jwt.strategy";
+import { ConfigModule } from "@nestjs/config";
+
+@Module({
+    controllers: [AuthenticationController],
+    imports: [
+        ConfigModule.forRoot({
+            envFilePath: `.env.${process.env.NODE_ENV}`,
+        }),
+        UsersModule,
+        PassportModule.register({ defaultStrategy: "jwt", session: false }),
```

Comment:
Do you need to register a default strategy? will be cleaner if you don't need it

```suggestion
        PassportModule,
```

Reply:
was added when troubleshooting the JwtService issue, works without it!